nox/virtualenv.py

```diff
@@ -137,7 +146,12 @@ def _resolved_interpreter(self):
             return self._resolved
 
         if self.interpreter is None:
-            self._resolved = sys.executable
+            if not _FROZEN:
+                self._resolved = sys.executable
+                return self._resolved
+
+            # Nox was frozen with PyInstaller
```

Comment:
Since this is the "rare" case I'd prefer it to be in the "early-exit-for-a-special-case" block.

Reply:
Ok